api/api_keys/permissions.py

```diff
@@ -0,0 +1,20 @@
+from contextlib import suppress
+
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework.permissions import BasePermission
+
+from organisations.models import Organisation
+
+
+class IsOrganisationAdmin(BasePermission):
```

Comment:
I'm not sure why this permission class exists in the `api_keys` module?

Reply:
Yeah - I think we should

Reply:
Yeah, I agree, maybe you can follow what I did for the `NestedEnvironmentPermissions` class?

Reply:
You mean  like this: https://github.com/Flagsmith/flagsmith/pull/1140/commits/511c0f64588069a82318eb103607432852bf4114?